api/customAttributes.js

```diff
@@ -38,7 +38,7 @@ module.exports = function(app) {
     const retObj = {
       breadcrumbs,
       parentsElection,
-      freightYear: 2018,
+      freightYear: 2020,
```

Comment:
just double-checking here: is 2020 actually the latest "real" year? I'm just wondering why we are missing 2019, and want to be extra certain.